addons/web_editor/static/src/components/media_dialog/media_dialog.scss

```diff
@@ -31,7 +42,21 @@
     }
 
     .o_we_load_more {
+        top: calc(var(--footer-top) - #{$bottom-bar-height} - #{$modal-inner-padding});
+        width: calc(var(--footer-width) - 2 * #{$modal-inner-padding});
+        height: $bottom-bar-height;
         scroll-margin: $modal-inner-padding;
+
+        &.o_can_scroll {
+            z-index: 2;
+            pointer-events: none;
+
+            > .o_scroll_attachments {
+                width: 36px;
+                height: 36px;
+                pointer-events: all;
+            }
+        }
```

Comment:
Looking at the lines, nothing inspire me good.
I am not a css expert so I'd like to discuss it tomorrow it will be easier.

Wouldn't it have been easier to have kept the structure we had before for the "load more" button and the "all loaded" text? Eg not floating with some css to fix the floating.
And just the "scroll arrow" button which would have been a floating element which wouldn't have taken all the width but just its own space?

And we wouldn't need all the css about difficult / weird `top` calculation but just something like
```css
bottom: 20px;
    left: 50%;
    position: absolute!important;
```